doc/src/reference/integration_status.md

```diff
@@ -0,0 +1,75 @@
+# Integration Status
+
+This section gives a status of libmicrovmi's integration into
+other applications, libraries, or based on _virtual machine introspection_
+
+## LibVMI
+
+- [project](https://github.com/libvmi/libvmi)
+- [issue](https://github.com/Wenzel/libmicrovmi/issues/137)
+- [fork](https://github.com/Wenzel/libvmi/tree/libmicrovmi)
+    - Note: use the `libmicrovmi` branch
```

Comment:
I think this would look a little better:
```suggestion
- [fork](https://github.com/Wenzel/libvmi/tree/libmicrovmi) (Note: use the `libmicrovmi` branch)
```